swagger_server/controllers/operand_controller.py

```diff
@@ -48,4 +68,12 @@ def put_operand(operand_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    if not payload:
+        raise ValidationError(400, 'Couldn\'t parse JSON POST body.')
+    if not payload.get('id'):
+        raise ValidationError(400, 'No id field specified')
+    try:
+        operator = Operand.update(payload).where(id=operand_id).execute()
+    except Exception as exc:
+        raise ValidationError(400, 'Expression not found!')
+    return flask.jsonify({'id': operand_id})
```

Comment:
Let's jsonify the object here as well. It'll probably look really similar to get_operand's jsonify call.